common/religion/doctrines/wc_doctrines_religions.txt

```diff
@@ -58,6 +58,7 @@
 			hostility_override_tenet_life_syncretism = 1
 			
 			#Pure evil religions
+			hostility_override_intolerance_hostility_doctrine = 3
```

Comment:
I would extend it to other religious families to make sure everyone considers the Hakkari to be evil.

Reply:
done